pycytominer/cyto_utils/cells.py

```diff
@@ -117,6 +132,9 @@ def __init__(
         # Throw an error if both subsample_frac and subsample_n is set
         self._check_subsampling()
 
+        # Confirm that the input fields of view is valid
+        self.fields_of_view = check_fields_of_view_format(self.fields_of_view)
```

Comment:
nice! I like this attribute assigned in this way. We can also do as suggested, and then drop line 111 where you're originally assigning the attribute. Super minor, but slightly more pythonic IMO
```suggestion
        self.fields_of_view = check_fields_of_view_format(fields_of_view)
```

Reply:
Done - https://github.com/cytomining/pycytominer/pull/129/commits/a8d20b2cc5e929736160bdc3ecff37a4aca1a10d